src/Tools/AI Test Toolkit/src/Logs/AITTestMethodLinesCompare.Page.al

```diff
@@ -0,0 +1,216 @@
+                    end;
+                }
+                field(BaseVersion; this.BaseVersion)
+                {
+                    Caption = 'Base Version';
+                    ToolTip = 'Specifies the Base version to compare to.';
+                    ApplicationArea = All;
+
+                    trigger OnValidate()
+                    begin
+                        UpdateVersionFilter();
+                    end;
+                }
+            }
+
+            group("Version Comparison")
+            {
+                Caption = 'Version Comparison';
+                grid(Summary)
+                {
+                    group("Summary Captions")
+                    {
+                        ShowCaption = false;
+                        label(NoOfTests)
+                        {
+                            Caption = 'No. of Tests';
+                            Tooltip = 'Specifies the number of tests in this Line';
+                            ApplicationArea = All;
+                        }
+                        label(NoOfTestsPassed)
+                        {
+                            Caption = 'No. of Tests Passed';
+                            ToolTip = 'Specifies the number of tests passed in the version.';
+                            ApplicationArea = All;
+                        }
+                        label(NoOfTestsFailed)
+                        {
+                            Caption = 'No. of Tests Failed';
+                            ToolTip = 'Specifies the number of tests that failed in the version.';
+                            ApplicationArea = All;
+                        }
+                        label(NoOfOperations)
+                        {
+                            Caption = 'No. of Operations';
+                            ToolTip = 'Specifies the number of operations in the version.';
+                            ApplicationArea = All;
+                        }
+                        label(TotalDuration)
+                        {
+                            Caption = 'Total Duration (ms)';
+                            ToolTip = 'Specifies Total Duration of the AIT for this role for the version.';
+                            ApplicationArea = All;
+                        }
+                    }
+                    group("Latest Version")
+                    {
+                        field("No. of Tests"; Rec."No. of Tests")
+                        {
+                            Tooltip = 'Specifies the number of tests in this Line';
+                            ApplicationArea = All;
+                            ShowCaption = false;
+                        }
+                        field("No. of Tests Passed"; Rec."No. of Tests Passed")
+                        {
+                            ApplicationArea = All;
+                            Style = Favorable;
+                            ToolTip = 'Specifies the number of tests passed in the current Version.';
```

Comment:
Applies to multiple fields:
```suggestion
                            ToolTip = 'Specifies the number of tests passed in the current version.';
```

If Version should be capitalized, then this needs to be adjusted in the labels above as well.